task-submission-service/src/main/resources/application.yml

```
@@ -0,0 +1,43 @@
+
+file:
+  upload-dir: ${FILE_UPLOAD_DIR:./uploads/submission/}
+
+eureka:
+  instance:
+    prefer-ip-address: true
+  client:
+    fetch-registry: true
+    register-with-eureka: true
+    serviceUrl:
+      defaultZone: http://localhost:8070/eureka/
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)